flows/aws/interactive-workflow/interactive-workflow.py

```diff
@@ -23,8 +23,8 @@
 ]
 
 
-class CreateArtifact(RunInput):
-    create_artifact: bool = Field(description="Would you like to create an artifact?")
+class userApproval(RunInput):
```

Comment:
Class names should follow the [CapWords convention](https://peps.python.org/pep-0008/#class-names).